### DISCUSSION TITLE
ebus adapter v5 PCB mount

### DISCUSSION THREAD

**Discussion by KrzysiekZZZ** [john30/ebusd#1058] [General]
Hi guys,
anyone have working solution how to mount the ebus adapter v5 on din rail, maybe customer 3d printed case or other solution ? 

Many thanks.
Chris

**Comment by andrewt27** [comment]
Try to find something like this.
https://botland.store/cases-casings/10131-modular-case-kradex-z103j-88x34x62mm-for-mounting-on-din-rail-light-colored-5905275012343.html
![modular-case-kradex-z103j-88x34x62mm-for-mounting-on-din-rail-light-colored](https://github.com/john30/ebusd/assets/36766135/b1234c41-96c3-4225-8c5c-aab30043d62b)

You can mount pcb to this case by "glue gun" or two side adhesive foam tape.

**Comment by KrzysiekZZZ** [comment]
Thanks.
I've ordered print of : https://www.thingiverse.com/thing:6000890 and mounted to separate din mount.
